_sass/base/_typography.scss

```diff
@@ -204,6 +205,28 @@ hr {
 
 center p a {
 	&:hover{
-		border-bottom: 2px solid _palette(accent);
+		border-bottom: 2px solid _palette(orange);
+	}
+}
+
+.contact a {
+	color: _palette(black);
+	&:hover {
+		color: _palette(orange);
+	}
+}
+
+a.toggle {
+	&::before {
+		&:hover {
+			color: _palette(orange) !important;
+		}
+	}
+}
+
+#prev_next a {
+	color: _palette(orange);
+	&:hover {
+		color: _palette(orange);
```

Comment:
This hover should be purple, now nothing is happening on the prev_next with hover.